bin/build-docker.sh

```diff
@@ -3,7 +3,14 @@
 VERSION=`jq -r ".version" package.json`
 SERIES=${VERSION:0:4}-latest
 
-cat package.json | grep -v electron > server-package.json
+if ! [[ $(which tsc) ]]; then
+    npm i -g typescript
+fi
+
+cat package.json > server-package.json
```

Comment:
Why was `grep -v electron` removed?

I suppose this is helping to reduce the size of the Docker image by not embedding electron dependencies.

Reply:
I added it back. It was causing problems by removing the `postinstall` script causing errors in testing. Later found out @alexpietsch added it a while ago, but it doesn't seam to be need. I ended up just getting rid of the reference to electron in the `postinstall` script.